abstract_specification.md

```diff
@@ -940,18 +938,15 @@ To facilitate distributed computation, read operations on foundational types acc
 | -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `IofN`         | Given I and N, read operations will return the Ith partition of N approximately equal size partitions. Partition boundaries will be stable for any given N and array or dataframe. |
 
-### SOMABatchFormat
+### SOMASparseNDArrayRead
 
-Array read operations can return results in a variety of formats. The `SOMABatchFormat` format indicates the format encoding.
+`SparseNDArray` `read` operations can return results in a variety of formats. The `SOMASparseNDArrayRead` type is an intermediate result type that allows the client to choose the encoding format that will be used to return the result data.
 
-| Batch format   | Description                                                                                            |
-| -------------- | ------------------------------------------------------------------------------------------------------ |
-| `dense`        | Return the coordinates of the slice (e.g., origin, shape) and an Arrow Tensor containing slice values. |
-| `coo`          | Return an `Arrow.SparseCOOTensor`                                                                      |
-| `csr`          | Return an `Arrow.SparseCSRTensor`                                                                      |
-| `csc`          | Return an `Arrow.SparseCSCTensor`                                                                      |
-| `record-batch` | Return an `Arrow.RecordBatch` containing COO-encoded coordinates and values.                           |
-| `table`        | Return an `Arrow.Table` containing COO-encoded coordinates and values.                                 |
+| Format         | Description                                                                                   |
+|----------------|-----------------------------------------------------------------------------------------------|
+| `dense`        | Return an iterator of `Arrow Tensor`s containing slice values.                                |
+| `coos`         | Return an iterator of `Arrow.SparseCOOTensor`s containing COO-encoded coordinates and values. |
+| `table`        | Return an iterator of `Arrow.Table`s containing COO-encoded coordinates and values.           |
```

Comment:
Not sure if csr, csc, and record-batch should be removed from the spec, but as they're no longer in the tiledbsoma impl, I removed them for now. If intended to remain, perhaps we should indicate that implementations can optionally implement formats.

Reply:
They need to be removed everywhere & it's a mistake on my part if I missed removing them from anywhere.

Reply:
Thanks for confirming. Will stick with the change as is; no other occurrences exist.

Reply:
Actually, `record-batch` is still in the core code so reinstated here